.tool-versions

```diff
@@ -6,4 +6,3 @@ helm 3.10.3
 zig 0.10.1
 golangci-lint 1.51.2
 shellspec 0.28.1
-codecgen 1.2.10
```

Comment:
Just curious, why was this needed?

Reply:
i added it mistakenly because it seemed like all our tools were listed here. i'm not sure what this file is intended to do

Reply:
ahh... maybe we should add this to the readme somewhere; `.tool-versions` is the file for asdf managed dev tools https://asdf-vm.com/ to help ensure version consistency, but it also only works for supported packages